client/sites-dashboard/components/sites-table.tsx

```diff
@@ -48,6 +52,16 @@ export function SitesTable( { className, sites }: SitesTableProps ) {
 				</Row>
 			</thead>
 			<tbody>
+				{ isLoading &&
+					Array( N_LOADING_ROWS )
+						.fill( null )
+						.map( ( _, i ) => (
+							<SitesTableRowLoading
```

Comment:
This is generating a React warning about a missing key. One of the few times I think it's ok to just use the array index as the key I think :)

```suggestion
							<SitesTableRowLoading
								key={ i }
```

Reply:
Nice catch! I didn't check the console, and I always forget the key.